Gemfile.lock

```diff
@@ -209,60 +209,60 @@ GEM
       sprockets-rails
       tilt
     semantic_range (2.3.0)
-    spring (2.1.0)
+    spring (2.1.1)
     spring-watcher-listen (2.0.1)
       listen (>= 2.7, < 4.0)
       spring (>= 1.2, < 3.0)
     sprockets (4.0.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
-    sprockets-rails (3.2.1)
+    sprockets-rails (3.2.2)
       actionpack (>= 4.0)
       activesupport (>= 4.0)
       sprockets (>= 3.0.0)
     thor (1.0.1)
     thread_safe (0.3.6)
     tilt (2.0.10)
-    tinymce-rails (5.4.1)
+    tinymce-rails (5.6.1)
       railties (>= 3.1.1)
-    trestle (0.9.3)
+    trestle (0.9.4)
       activemodel (>= 4.2.0)
       kaminari (>= 1.1.0)
       railties (>= 4.2.0)
       sprockets-rails (>= 2.0.0)
-    trestle-active_storage (3.0.0)
+    trestle-active_storage (3.0.1)
       rails (>= 5.2, < 7)
       trestle (~> 0.9.0, >= 0.9.3)
-    trestle-auth (0.4.0)
+    trestle-auth (0.4.1)
       bcrypt (~> 3.1.7)
-      trestle (~> 0.9.0)
+      trestle (~> 0.9.0, >= 0.9.3)
     trestle-tinymce (0.3.0)
       tinymce-rails (~> 5.1)
       trestle (~> 0.9.0)
     turbolinks (5.2.1)
       turbolinks-source (~> 5.2)
     turbolinks-source (5.2.0)
-    tzinfo (1.2.7)
+    tzinfo (1.2.8)
       thread_safe (~> 0.1)
     unf (0.1.4)
       unf_ext
     unf_ext (0.0.7.7)
-    warden (1.2.8)
-      rack (>= 2.0.6)
-    web-console (4.0.4)
+    warden (1.2.9)
+      rack (>= 2.0.9)
+    web-console (4.1.0)
       actionview (>= 6.0.0)
       activemodel (>= 6.0.0)
       bindex (>= 0.4.0)
       railties (>= 6.0.0)
-    webpacker (5.1.1)
+    webpacker (5.2.1)
       activesupport (>= 5.2)
       rack-proxy (>= 0.6.1)
       railties (>= 5.2)
       semantic_range (>= 2.3.0)
     websocket-driver (0.7.3)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
-    zeitwerk (2.4.0)
+    zeitwerk (2.4.2)
```

Comment:
### [zeitwerk](https://github.com/fxn/zeitwerk)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.4.0 → 2.4.2

[_[change-log](https://github.com/fxn/zeitwerk/blob/master/CHANGELOG.md), [source-code](https://github.com/fxn/zeitwerk)_]





<details>
<summary>Commits</summary>

A change of **12** commits. See the full changes on [the compare page](https://github.com/fxn/zeitwerk/compare/v2.4.0...v2.4.2).

These are the first 10 commits:
- (90620a1) [fixes typo](https://github.com/fxn/zeitwerk/commit/90620a189d41200720e7a0c9f1e0cd8f7b793df8)
- (8167a8d) [Use __dir__ in collapse examples [Closes #131]](https://github.com/fxn/zeitwerk/commit/8167a8dcd6b6dcc5c625d701836282f9c19ef7ca)
- (f892e80) [Add truffleruby-head to CI](https://github.com/fxn/zeitwerk/commit/f892e800e5dfdc879bf046922c89703bb5da7dae)
- (1790e82) [Merge pull request #132 from bjfish/truffleruby-ci](https://github.com/fxn/zeitwerk/commit/1790e82c579e118522c181e37c3f4870e17deb56)
- (627482a) [removes truffleruby-head from allowed CI failures 🎉](https://github.com/fxn/zeitwerk/commit/627482af33c895c7199b261e78f4d8037b01938e)
- (60021d9) [RBS notation for methods and attributes](https://github.com/fxn/zeitwerk/commit/60021d90af973d98ee19e9634a552b25c39320cc)
- (cce3c39) [Use __send__ instead of send](https://github.com/fxn/zeitwerk/commit/cce3c392e0de92569529bffac46e26cc0e716805)
- (26f232e) [Merge pull request #140 from mfilej/send](https://github.com/fxn/zeitwerk/commit/26f232e21c8faad5d5d1143dcc732845ff6db569)
- (034ae30) [version 2.4.1](https://github.com/fxn/zeitwerk/commit/034ae30d73247b8dda7df2992903ce560cd7f47f)
- (c716367) [Implements Zeitwerk::Loader#on_load](https://github.com/fxn/zeitwerk/commit/c71636797544a649eb6cb40bcc516736bb57d3d4)


</details>